### ISSUE TITLE
Cloning of data sets

### ISSUE THREAD

**Issue by frankRuehle** [rdmorganiser/rdmo#474]
It would be nice to have functionality for easy cloning of data sets / projects in RDMO.
[This issue is from 7th RDMO community meeting on March 2, 2022]

**Comment by jochenklar** [comment]
This is also a duplicate of https://github.com/rdmorganiser/rdmo/issues/474.

**Comment by jochenklar** [comment]
Hi @langeipk , we will add the possibility to

* "apply" given answers to other datasets/tabs,
* to clone datasets/tabs, and
* to "import" values from other projects as answers for specific questions

in RDMO 2.3.0 (the version after the next one).

**Comment by jochenklar** [comment]
My idea was, to have those functions on every question (when it makes sense), but we can also think about a switch to control that. However, this would add yet another piece of complexity to "content creation" in RDMO, which I try to avoid.

**Comment by langeipk** [comment]
I agree, increased complexity include feature to make fields clonable in the template. Maybe as sub-issue after the clone feature is rolled out. Happy to have this in 2.3.0. Thanks!

**Comment by jochenklar** [comment]
The question if this feature should be controllable in the settings or the catalog is still out, but the feature will be released soon in RDMO 2.3.0.
